src/frontend/admin/src/components/presentational/table/TableComponent.tsx

```diff
@@ -151,8 +158,7 @@ export function TableComponent<T>({ enableEdit = true, ...props }: Props<T>) {
             footerRowSelected: (count) =>
               intl.formatMessage(tableTranslations.rowsSelected, { count }),
           }}
-          onPaginationModelChange={(size) => setPageSize(size.pageSize)}
-          pageSizeOptions={[5, 10, 25, 50, 75]}
+          pageSizeOptions={[20]}
```

Comment:
What do you think to define default page size in a global constant as this value is used in several modules (`TableComponent` and `useTableResourcePagination`)